poradnia/events/tests/test_management_commands.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import print_function
+
+from datetime import timedelta
+from unittest import TestCase
```

Comment:
dzięki, nie wiedziałem o tym, wyjaśniałoby to problemy jakie miałem przy testowaniu :)